index.html

```diff
@@ -66,6 +257,41 @@ <h1>I’m Belal Gamal Glad to see you!</h1>
             </li>
           </ul>
         </div>
+        <button type="button" class="btn">Get my resume</button>
+        <div class="skills-list">
+          <div class="lang">
+            <div class="skills-list-style">
+              <h4>Languages</h4>
+              <img src="./images//down-arrow.svg" alt="arrow" />
+            </div>
+            <div class="languages-list">
+              <ul>
+                <li>
+                  <img src="./images/Ellipse 1.png" alt="javascript-icon" />
+                  <h5>Javascript</h5>
+                </li>
+                <li>
+                  <img src="./images/html-icon.jpg" alt="html-icon" />
+                  <h5>HTML</h5>
+                </li>
+                <li>
+                  <img src="./images/css-icon.png" alt="css-icon" />
+                  <h5>CSS</h5>
+                </li>
+              </ul>
+            </div>
+          </div>
+          <div class="skills-list-style">
+            <h4>Frameworks</h4>
+            <img src="./images//right-arrow.svg" alt="arrow" />
+          </div>
+          <hr />
+          <div class="skills-list-style">
+            <h4>Skills</h4>
+            <img src="./images//right-arrow.svg" alt="arrow" />
+          </div>
```

Comment:
- Here you did a great job, your design looks very nice but makes sure this section is visible in the mobile version. as I show you in the picture below I'm only able to see the frameworks but not the skills. kindly ensure that you fix this issue :grinning:  :grinning:  :grinning: 
## Your design 
 
![image](https://user-images.githubusercontent.com/95297251/191987120-bc3939b3-c44d-4e37-95f6-8a1a0398e672.png)

## Figma design
 
![image](https://user-images.githubusercontent.com/95297251/191989175-fee409ac-a908-46a4-bc49-be15f1256dae.png)

Reply:
@MaganaAsiati it is visible on my laptop
![Screenshot from 2022-09-23 17-05-39](https://user-images.githubusercontent.com/32738452/191992761-da45b98a-2ec6-4341-b1f1-1c156c8353ad.png)